forge/db/models/Notification.js

```diff
@@ -89,6 +89,25 @@ module.exports = {
                         count,
                         notifications: rows
                     }
+                },
+                markNotificationsAsRead: async ({ read = true, ids = [] }, user) => {
```

Comment:
```suggestion
                updateNotificationState: async ({ read = true, ids = [] }, user) => {
```

Do we think, before this is committed, we should adjust this function name?